src/sources/socket/mod.rs

```diff
@@ -347,7 +347,6 @@ mod test {
 
     #[test]
     fn tcp_shutdown_infinite_stream() {
-        crate::test_util::trace_init();
```

Comment:
Is this really intended?

Reply:
Yes, I added it in the previous commit and forgot to remove it.

Reply:
Oh, got it.
We probably should add this to every test out there - at least that's how I understand it. The reason some tests don't have it seem to be it was added after the tests.
Not sure though.